rest-fetcher/src/main/java/com/ackee/versionupdatehandler/RestVersionFetcher.kt

```diff
@@ -0,0 +1,70 @@
+package com.ackee.versionupdatehandler
+
+import com.ackee.versioupdatehandler.VersionFetcher
+import com.ackee.versioupdatehandler.model.BasicVersionsConfiguration
+import com.ackee.versioupdatehandler.model.DefaultVersionsConfiguration
+import com.ackee.versioupdatehandler.model.VersionsConfiguration
+import com.google.gson.FieldNamingPolicy
+import com.google.gson.GsonBuilder
+import com.google.gson.JsonObject
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.GET
+
+/**
+ * Class that fetches version configuration from rest api
+ *
+ *
+ * Api is specified by base url and is expected to be at address /app_version
+ *
+ * @author David Bilik [david.bilik@ackee.cz]
```

Comment:
remove this javadoc pls, its not true at all anymore

Reply:
5ff6590de61b394853088024f80a06efab417818